init.lua

```diff
@@ -425,4 +425,4 @@ minetest.register_node("prefab:concrete_catwalk",{
 	},
 	groups={cracky=2},
 	drop = "prefab:concrete_catwalk",
-})
+})
```

Comment:
If this were Minetest code style guidelines, a new line should be added to the end of the file.